core/math/a_star_grid_2d.h

```diff
@@ -91,6 +91,10 @@ class AStarGrid2D : public RefCounted {
 		real_t abs_g_score = 0;
 		real_t abs_f_score = 0;
 
+		Vector2 flow_direction;
+		uint32_t value;
```

Comment:
This value seems like an internal thing, only used during the function calculation. Maybe remove it and use an array you allocate only during the function.

Reply:
Done, I think, please check again